From 55653d91626d7617a43dc9870380b41f2e4b5ccb Mon Sep 17 00:00:00 2001
From: Andreas Cadhalpun <Andreas.Cadhalpun@googlemail.com>
Date: Sun, 26 Apr 2015 20:05:40 +0200
Subject: [PATCH 2/2] tests: return 0 upon success

---
 tests/testframework.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/testframework.c b/tests/testframework.c
index a38851c..a889560 100644
--- a/tests/testframework.c
+++ b/tests/testframework.c
@@ -36,7 +36,7 @@ int unittest_summary(){
   fprintf(stderr, "UNIT TESTs succeeded:\t %s%i/%i\033[0m\n",
           units_failed>0 ? "\033[1;31m" : "\033[1;32m",
           units_success, units_success + units_failed);
-  return units_failed==0;
+  return units_failed!=0;
 
 }
 
-- 
2.1.4

